apps/studio/src/server/modules/page/page.router.ts

```diff
@@ -2,16 +2,25 @@ import type { IsomerSchema } from "@opengovsg/isomer-components"
 import { schema } from "@opengovsg/isomer-components"
 import { TRPCError } from "@trpc/server"
 import Ajv from "ajv"
-import { z } from "zod"
+import { isEqual } from "lodash"
```

Comment:
nit: unsure if this can be treeshaken. Suggest using lodash's explicit export.

```suggestion
import isEqual from "lodash/isEqual"
```

Reply:
done